torchgeo/datasets/utils.py

```diff
@@ -255,6 +258,50 @@ def intersects(self, other: "BoundingBox") -> bool:
         )
 
 
+def disambiguate_timestamp(date_str: str, format: str) -> Tuple[float, float]:
+    """Disambiguate partial timestamps.
+
```

Comment:
Can you add a brief description of what is going on in this method (and why) for posterity? This looks like something I would quickly forget about and question the sanity of.